modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotMessageCounter.java

```diff
@@ -102,11 +112,10 @@ public void run() {
                         } catch (ConnectionException e) {
                             log.error("Error while sending slot deletion safe zone update", e);
                         }
-
                     }
                 }
             }
-        }, 4000, 3000);
+        }, SLOT_SUBMIT_TIMEOUT, SLOT_SUBMIT_TASK_EXEC_PERIOD);
```

Comment:
why 4000 and 3000 , why can't it be 4500, 3500 ? (basis for the question is are these number chosen based on educated guess or just random?